src/app/pages/SignUpPage/SignUpPage.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import Typography from '../../components/Typography/Typography';
+import { Link } from 'react-router-dom';
+import styles from './SignUpPage.module.css';
+
+export default function SignUpPage(): JSX.Element {
+  return (
+    <main className={styles.signup}>
+      <h2>
+        <Typography size="l">Anmeldung</Typography>
+      </h2>
```

Comment:
Dude, you already made Typography to be cool enough, no need for extra elements :)

Reply:
Dang you're right, I forgot that my Typography is on steroids!